service/lexruntimeservice/go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/aws/aws-sdk-go-v2/service/lexruntimeservice
 
-go 1.15
+go 1.14
```

Comment:
The go version is inconsistent in the PR (benchmark go.mod has go1.15, while other places have go1.14). Don't have to address it in this PR. 

But seems like you need to do go mod tidy (assuming you are on go 1.15).

Reply:
thanks forgot to regenerate the clients which is where this came in.